CHANGES.txt

```diff
@@ -4,6 +4,11 @@ Next release
 - Fixed a failing unittest caused by differing mimetypes on various
   OS platforms. See https://github.com/Pylons/pyramid/issues/1405
 
+- Overall improvments for the ``proutes`` command. Added ``--format`` and
+  ``--glob`` arguments to the command, introduced the ``method``
+  column for displaying available request methods, and improved the ``view``
+  output by showing the module instead of just ``__repr__``.
```

Comment:
Yeah, I didn't know if I should link #1488 or this specific backport one, what do you think?